src/Compilers/CSharp/Test/Semantic/Semantics/UsingStatementTests.cs

```diff
@@ -1189,50 +827,23 @@ static void Main()
                 );
         }
 
-        [Fact]
-        public void UsingPatternExtensionMethodInParameterOnStruct()
-        {
-            var source = @"
-struct S1
-{
-}
-
-static class C1 
-{
-   public static void Dispose(in this S1 s1) { }
-}
-
-class C2
-{
-    static void Main()
-    {
-       using (S1 s = new S1())
-       {
-       }
-    }
-}";
-
-            var compilation = CreateCompilation(source).VerifyDiagnostics();
-        }
-
         [Fact]
         public void UsingPatternWithParamsTest()
```

Comment:
>UsingPatternWithParamsTest [](start = 20, length = 26)

Please add a test for extension method `Dispose` with additional `params` argument.

Reply:
Added, which exposed another lowering failure, which should now also be fixed.